python/cudf/cudf/io/parquet.py

```diff
@@ -534,6 +534,7 @@ def read_parquet(
     filters=None,
     row_groups=None,
     use_pandas_metadata=True,
+    allow_mismatched_pq_schemas=False,
```

Comment:
Introducing a function parameter in middle will become a breaking change, can we please add it to the end since pandas doesn't have this parameter?

Reply:
Thanks for the suggestion, all done!